### PR TITLE
Multiple changes for version 0.1.1

### PR DESCRIPTION
**Pull request by Adrianzatreanu**

### CODE REVIEW

File: MANIFEST [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# file GENERATED by distutils, do NOT edit
+setup.py
+coala-decorators/Decorators.py
```
Comment (sils) [inline MANIFEST]: don't put generated files in the repo

Comment (Adrianzatreanu) [inline MANIFEST]: we dont need it? the manifest?

**Comment by Adrianzatreanu** [comment]
@sils1297 updated, feel free to merge ^^

**Comment by sils** [comment]
@rultor merge

**Pull request merged**